zoloto/marker.py

```diff
@@ -64,10 +64,7 @@ def orientation(self) -> Orientation:
 
     @cached_property
     def spherical(self) -> Spherical:
-        x, y, z = self._tvec
-        return Spherical(
-            rot_x=float(arctan2(y, z)), rot_y=float(arctan2(x, z)), dist=self.distance
```

Comment:
Suggestion: Now that distance calculations are done inside `Spherical`, perhaps it'd be nice to update `Marker.distance` to `return self.spherical.distance`? Save a duplicate implementation.

The spherical implementation feels easier to validate and prove than `linalg.norm`.

Reply:
https://github.com/RealOrangeOne/zoloto/pull/302/commits/4654ddc0218c20f9b8c797cfa62f795021566715